src/PrestaShopBundle/Service/HttpClient/SymfonyClient.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Service\HttpClient;
+
+use Exception;
+use PrestaShop\CircuitBreaker\Contract\ClientInterface;
+use PrestaShop\CircuitBreaker\Exception\UnavailableServiceException;
+use PrestaShop\CircuitBreaker\Exception\UnsupportedMethodException;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class SymfonyClient implements ClientInterface
```

Comment:
Hi @lartist, 
I was wondering if we should not put this in the adapters ?

Reply:
I think that does not make sense, IIRC adapters should only be a glue between the legacy and Symfony? There's no legacy here so It should stay inside the prestashop bundle IMO.